tests/console/gpg.pm

```diff
@@ -115,7 +116,7 @@ EOF
     # the random probable primes p and q for RSA.
     #
     # Please see bsc#1165902#c40 that RSA 4096 can be accepted even in FIPS mode
-    if (get_var('FIPS') || get_var('FIPS_ENABLED') && ($key_size == '1024')) {
+    if ((get_var('FIPS') || get_var('FIPS_ENABLED') || get_var('FIPS_ENV_MODE')) && ($key_size == '1024')) {
```

Comment:
I don't understand this.

Reply:
For some reason, there are several variables for FIPS. As I don't want to change behaviour much, I have added `FIPS_ENV_MODE` here, which is also used in line 64. That's the variable that I have added to SLE Micro/MicroOS tests.